src/development/tools/devtools/performance.md

```diff
@@ -18,28 +18,39 @@ your application. It consists of three parts, each increasing in granularity.
 * Timeline events chart
 * CPU profiler
 
-{{site.alert.note}}
+{{site.alert.secondary}}
   **Use a profile build of your application to analyze performance.**
-  Frame rendering times are not indicative of release performance
+  Frame rendering times aren't indicative of release performance
   unless your application is run in profile mode.
 {{site.alert.end}}
 
 The performance view also supports importing and exporting of
-data snapshots. For more information, see the [Import and export][] section.
+data snapshots. For more information,
+check out the [Import and export][] section.
+
+## What is a frame in Flutter?
+
+Flutter is designed to render its UI at 60 frames per second
+(fps), or 120 fps on devices capable of 120Hz updates.
+This means that, approximately every 16ms, the UI updates
+to reflect animations, or other changes to the UI. A frame
+that takes longer than 16ms to render can cause jank 
+(jerky movement).
```

Comment:
**issue:** The graf doesn't answer the question posed in the heading.
```suggestion
Flutter renders its UI at least 60 times each second.
Each render is called a _frame_. Flutter can render 
120 frames per second (fps) on displays that refresh at 120Hz.
This means Flutter updates the UI about every 16ms
to reflect animations or other changes to the UI. If a frame
renders in more than 16ms, it can cause jank, or jerky movement,
on the device display.
```

Reply:
I wouldn't say "it renders" as opposed to "it's designed to render". In other words, it MIGHT take longer, in which case it's the programmers fault.